### PR TITLE
[feature] Segment analytics implementation

### PR DESCRIPTION
**Pull request by alex-deaconu**
New CH version
Removed event tracking; moved pageview tracking to CH

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@olegrise please review. Thanks!

File: src/com/risevision/ui/client/common/controller/CommonHeaderController.java [hunk 1]
```diff
@@ -35,7 +35,15 @@ public static int getContentHeight() {
...
+		trackPageviewNative(contentId);
+	}
+	
 	private static native void updateCompanyIdNative(String companyId) /*-{
 		$wnd.angular.element($wnd.document.getElementsByTagName("common-header")).scope().updateCompanyId(companyId);
 	}-*/;
+	
+	private static native void trackPageviewNative(String contentId) /*-{
+		$wnd.angular.element($wnd.document.getElementsByTagName("common-header")).scope().trackPageview(contentId);
+	}-*/;
```
Comment (olegrise) [inline src/com/risevision/ui/client/common/controller/CommonHeaderController.java]: Wow, this is some ninja style coding :sunglasses:

File: war/Rdn2ui.html [hunk 2]
```diff
@@ -67,27 +57,6 @@
 	<!--[if lt IE 9]>
     	<script type="text/javascript" src="scripts/json2.js"></script>
 	<![endif]-->
-    
-<!--	<script type="text/javascript" charset="utf-8">-->
-<!--	  var is_ssl = ("https:" == document.location.protocol);-->
-<!--	  var asset_host = is_ssl ? "https://s3.amazonaws.com/getsatisfaction.com/" : "http://s3.amazonaws.com/getsatisfaction.com/";-->
-<!--	  document.write(unescape("%3Cscript src='" + asset_host + "javascripts/feedback-v2.js' type='text/javascript'%3E%3C/script%3E"));-->
```
Comment (olegrise) [inline war/Rdn2ui.html]: @alex-deaconu this is not related to segment analytics. Is it?

**Comment by olegrise** [comment]
:+1: jut one comment related to removing getsatisfaction script.

**Comment by alex-deaconu** [comment]
@olegrise I've refactored the spaghetti code, good call. With regards to the getSatisfaction script, that was a test we performed many years ago in the RVA, I just removed the commented out code. 

Thanks.
